.travis.yml

```diff
@@ -1,3 +1,10 @@
+matrix:
+  include:
```

Comment:
@syphernl works :tada: 

```
vagrant@local:~$ cat /etc/*-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04 LTS"
NAME="Ubuntu"
VERSION="16.04 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
UBUNTU_CODENAME=xenial
vagrant@local:~$ cat /etc/apt/sources.list.d/mariadb.list
deb http://ftp.nluug.nl/db/mariadb/repo/10.1/ubuntu xenial main
vagrant@local:~$ sudo systemctl status mysql
● mariadb.service - MariaDB database server
   Loaded: loaded (/lib/systemd/system/mariadb.service; enabled; vendor preset: enabled)
  Drop-In: /etc/systemd/system/mariadb.service.d
           └─migrated-from-my.cnf-settings.conf
   Active: active (running) since Thu 2017-07-27 10:54:04 CEST; 18min ago
  Process: 6760 ExecStartPost=/bin/sh -c systemctl unset-environment _WSREP_START_POSITION (code=exited, status=0/SUC
  Process: 6756 ExecStartPost=/etc/mysql/debian-start (code=exited, status=0/SUCCESS)
  Process: 6570 ExecStartPre=/bin/sh -c [ ! -e /usr/bin/galera_recovery ] && VAR= ||   VAR=`/usr/bin/galera_recovery`
  Process: 6566 ExecStartPre=/bin/sh -c systemctl unset-environment _WSREP_START_POSITION (code=exited, status=0/SUCC
  Process: 6562 ExecStartPre=/usr/bin/install -m 755 -o mysql -g root -d /var/run/mysqld (code=exited, status=0/SUCCE
 Main PID: 6726 (mysqld)
   Status: "Taking your SQL requests now..."
   CGroup: /system.slice/mariadb.service
           └─6726 /usr/sbin/mysqld

Jul 27 10:54:04 local mysqld[6726]: 2017-07-27 10:54:04 139674489178368 [Note] InnoDB:  Percona XtraDB (http://www.pe
Jul 27 10:54:04 local mysqld[6726]: 2017-07-27 10:54:04 139674489178368 [Note] Plugin 'FEEDBACK' is disabled.
Jul 27 10:54:04 local mysqld[6726]: 2017-07-27 10:54:04 139673519257344 [Note] InnoDB: Dumping buffer pool(s) not yet
Jul 27 10:54:04 local mysqld[6726]: 2017-07-27 10:54:04 139674489178368 [Note] Server socket created on IP: '127.0.0.
Jul 27 10:54:04 local mysqld[6726]: 2017-07-27 10:54:04 139674489178368 [Note] Reading of all Master_info entries suc
Jul 27 10:54:04 local mysqld[6726]: 2017-07-27 10:54:04 139674489178368 [Note] Added new Master_info '' to hash table
Jul 27 10:54:04 local mysqld[6726]: 2017-07-27 10:54:04 139674489178368 [Note] /usr/sbin/mysqld: ready for connection
Jul 27 10:54:04 local mysqld[6726]: Version: '10.1.25-MariaDB-1~xenial'  socket: '/var/run/mysqld/mysqld.sock'  port:
Jul 27 10:54:04 local mysqld[6756]: Checking for corrupt, not cleanly closed and upgrade needing tables.
Jul 27 10:54:04 local systemd[1]: Started MariaDB database server.
```